dts/bindings/mtd/spi-nv-flash.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Where is this used?

Reply:
We want to use this for further spi flash. An example for that was in this PR before (something like https://github.com/zephyrproject-rtos/zephyr/pull/11278/files#diff-0ad92537b7c3e2041e8e2a6b9a30698b) - but it was tu much for this PR. So using of this will be the next step (subsequent PR)

Reply:
Let's remove it for now, and keep this to things used by what's in the code base.